libs/librepcb/common/fileio/transactionalfilesystem.cpp

```diff
@@ -68,7 +68,7 @@ TransactionalFileSystem::TransactionalFileSystem(const FilePath& filepath,
     if (restoreMode == RestoreMode::ASK) {
       QMessageBox::StandardButton btn = QMessageBox::question(
           0, tr("Restore autosave backup?"),
-          tr("It seems that the application was crashed the last time. "
+          tr("It seems that the application crashed the last time you opened this project. "
```

Comment:
Line looks a bit long, could you run clang-format? :wink:

Reply:
Done.